pkg/deployer/bluegreen.go

```diff
@@ -49,38 +51,38 @@ func (b BlueGreen) Deploy(config builder.Config) error {
 			continue
 		}
 
-		//Setup frigga with prefix
-		frigga.Prefix = tool.BuildPrefixName(b.AwsConfig.Name, b.Stack.Env, region.Region)
-
 		//select client
 		client, err := selectClientFromList(b.AWSClients, region.Region)
 		if err != nil {
 			return err
 		}
 
-		// Get All Autoscaling Groups
-		asgGroups := client.EC2Service.GetAllMatchingAutoscalingGroupsWithPrefix(frigga.Prefix)
+		////Setup frigga with prefix
+		frigga.Prefix = tool.BuildPrefixName(b.AwsConfig.Name, b.Stack.Env, region.Region)
 
-		//Get All Previous Autoscaling Groups and versions
-		prevAsgs := []string{}
-		prevInstanceIds := []string{}
-		prevVersions := []int{}
-		var prevInstanceCount builder.Capacity
-		for _, asgGroup := range asgGroups {
-			prevAsgs = append(prevAsgs, *asgGroup.AutoScalingGroupName)
-			prevVersions = append(prevVersions, tool.ParseVersion(*asgGroup.AutoScalingGroupName))
-			for _, instance := range asgGroup.Instances {
-				prevInstanceIds = append(prevInstanceIds, *instance.InstanceId)
-			}
-
-			prevInstanceCount.Desired = *asgGroup.DesiredCapacity
-			prevInstanceCount.Max = *asgGroup.MaxSize
-			prevInstanceCount.Min = *asgGroup.MinSize
-		}
-		b.Logger.Info("Previous Versions : ", strings.Join(prevAsgs, " | "))
+		//// Get All Autoscaling Groups
```

Comment:
It can be deleted.

Reply:
I deleted all the commented lines :) Thanks